### DISCUSSION TITLE
Alarm notification if the process/app dies

### DISCUSSION THREAD

**Discussion by harvetech** [netdata/netdata#14190] [Ideas]
Dear Community, 

I have discovered netdata and I really like it. However, I have rather a simple server that I want to monitor. I have 5 services that are running and  I need a way to setup email notifications if any one of the services is taking too many resources or if it dies. 

I have looked into the documentation and some tutorial but it is always about some calculation. In my case, there is no calculation. Either up or down. How can I make such an alarm? 

any guidance will be highly appreciated. 

Best Regards

**Comment by ilyam8** [comment]
Hi, @harvetech. I think the [apps.processes](https://github.com/netdata/netdata/tree/master/collectors/apps.plugin#appsplugin) chart can be used for this - 0 processes means an app is not running. Relevant [community forum topic](https://community.netdata.cloud/t/apps-groups-conf-and-exporter-when-process-is-not-running/2141).
